Update add-inline-manifests-using-file-store.md (#6937)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/services/add-inline-manifests-using-file-store.md

## Reference files in the file store using an expression

The expression `fileStore.getAsString()` can be used to fetch files in the File Store in a script or runtime input. 

:::important

> Using the file in a Shell step or as a runtime input does not add it to the **Referenced by** list, making the file prone to deletion even if it is actively used.

Reviewer: ```suggestion
Referencing files in a Shell step or as a runtime input does not add it to the **Referenced by** list, making the file prone to deletion even if it is actively used.
```

## Reference files in the file store using an expression

> The expression `fileStore.getAsString()` can be used to fetch files in the File Store in a script or runtime input. Using the file in a shell step or runtime input does not update the "referenced by" list for the referenced file in the expression, making the file prone to deletion even if it is actively used.

Reviewer: ```suggestion
The expression `fileStore.getAsString()` can be used to fetch files in the File Store in a script or runtime input. Using the file in a Shell step or as a runtime input does not add it to the **Referenced by** list, making the file prone to deletion even if it is actively used.
```
